source/unified-test-format/unified-test-format.rst

```diff
@@ -1006,6 +1006,12 @@ The structure of each object is as follows:
   executing `operations`_. If the array is empty, the test runner MUST assert
   that no events were observed on the client (excluding ignored events).
 
+- ``eventMatch``: Optional string.  Specifies how the ``events`` array is
+  matched against the observed events.  Valid values are ``exact``, meaning
+  that observed events after all specified events have matched MUST cause a
+  test failure, and ``prefix``, meaning that observed events after all
+  specified events have been matched MUST NOT cause a test failure.
```

Comment:
With respect to the [Defy Augury](https://github.com/mongodb/specifications#defy-augury) mantra, is it likely that additional matching strategies will be needed down the line? If not, I think this might make more sense as an `ignoreExtraEvents` boolean that defaults to `false`.

If that change is made, the only necessary `invalid` test would be `expectedEventsForClient-ignoreExtraEvents-type` with a non-boolean value.